app/src/test/java/ca/klapstein/baudit/Data/PhoneNumberTest.java

```diff
@@ -4,34 +4,66 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import static junit.framework.TestCase.fail;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 
 public class PhoneNumberTest {
     @Before
-    public void setUp() {
-    }
+    public void setUp() {}
 
     @After
-    public void tearDown() {
-    }
+    public void tearDown() {}
 
     @Test
-    public void test_PhoneNumberConstructor() {
-        PhoneNumber phoneNumber = new PhoneNumber("111-111-1111");
+    public void testPhoneNumberConstructor() {
+        PhoneNumber phoneNumber = new PhoneNumber("(780)4444444");
         assertNotNull(phoneNumber);
-        assertEquals(phoneNumber.getPhoneNumber(), "111-111-1111");
+        assertEquals("7804444444", phoneNumber.getPhoneNumber());
     }
 
     @Test
-    public void test_getPhoneNumber() {
+    public void testGetPhoneNumber() {
+        PhoneNumber phoneNumber = null;
+
+        try {
+            phoneNumber = new PhoneNumber("012-345-6789");
+        } catch(IllegalArgumentException e) {
+            fail();
+        }
+
+        assertNotNull(phoneNumber);
+        assertEquals("0123456789", phoneNumber.getPhoneNumber());
     }
 
     @Test
-    public void test_setPhoneNumberValid() {
+    public void testSetPhoneNumberValid() {
+        try {
+            PhoneNumber phoneNumber = new PhoneNumber("(780) 555 5555");
+            assertEquals("7805555555", phoneNumber.getPhoneNumber());
+            phoneNumber.setPhoneNumber("780-555-5556");
+            assertEquals("7805555556", phoneNumber.getPhoneNumber());
+            phoneNumber.setPhoneNumber("780 555-5557");
+            assertEquals("7805555557", phoneNumber.getPhoneNumber());
+            phoneNumber.setPhoneNumber("(780)-555-5558");
+            assertEquals("7805555558", phoneNumber.getPhoneNumber());
+        } catch (IllegalArgumentException e) {
+            fail();
+        }
+
     }
 
     @Test
-    public void test_setPhoneNumberInvalid() {
+    public void testSetPhoneNumberInvalid() {
+        boolean success = true;
```

Comment:
overly complex

could be simplifed to:

```java
 new PhoneNumber("[780]-555-5555");
```

This provides the same testing functionality.